src/tests/unit/handler.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const { handler } = require('../../handler')
+const chai = require('chai')
+const expect = chai.expect
+
+describe('Tests successful response when all query parameters are given', function () {
+  const event = {
+    query: {
+      query: 'journal',
+      year: 2021,
+      start: 1
+    }
+  }
+
+  describe('Test search response content', () => {
+    it('verifies response is success and has a empty body', async function () {
```

Comment:
The body is not checked that is empty and it is not empty.

Reply:
Now it checks that the body is a empty object

Reply:
expect(response.body).to.be.an('Object').that.is.empty or expect(response.body).to.be.empty